template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CmHostGroupRoleConfigProviderProcessor.java

```diff
@@ -35,6 +38,23 @@
     @Inject
     private List<CmHostGroupRoleConfigProvider> providers;
 
+    @Value("#{'${cb.cm.sharedComponents:}'.split(';')}")
+    private List<String> sharedComponentsList;
+
+    private Map<String, String> sharedComponents = new HashMap<>();
+
+    @PostConstruct
+    private void parseSharedComponents() {
+        sharedComponentsList.forEach(s -> {
+            try {
+                String[] split = s.split("=");
```

Comment:
I think it would be nice to see somewhere what are the configs

Reply:
Do you mean a log message?

Reply:
yep